### ISSUE TITLE
BFO:0000050

### ISSUE THREAD

**Issue by jamesamcl** [EBISPOT/ols4#371]
When I search for BFO:0000050 I get this

<img width="1592" alt="Screenshot 2023-05-26 at 02 00 07" src="https://github.com/EBISPOT/ols4/assets/454895/3d1a230d-c265-4b5e-b622-ae46f2cff03e">

it's defined by RO but RO is nowhere to be seen. This is because it's defined by RO but uses the BFO prefix.

**Comment by serjoshua** [comment]
<img width="1532" alt="Screen Shot 2023-06-06 at 1 49 29 PM" src="https://github.com/EBISPOT/ols4/assets/25520595/a11250e2-9f69-4d0b-a595-09b23544ea9e">

(maybe related, short form or name is truncated)

**Comment by henrietteharmse** [comment]
It is not immediately clear how to address this without causing a lot of noise for various ontologies since many of our ontologies do not use `owl:imports`.

**Comment by matentzn** [comment]
Is there no rdfs:isDefinedBy property for these terms in RO? If not, I can request it

**Comment by henrietteharmse** [comment]
No, not for `BFO_0000050`. For `BFO_0000054`, `BFO_0000066` and `BFO_0000067` it has `<rdfs:isDefinedBy rdf:resource="http://purl.obolibrary.org/obo/bfo.owl"/>` and for BFO_0000055 it has `<rdfs:isDefinedBy rdf:resource="http://purl.obolibrary.org/obo/iao.owl"/>` 

It will be helpful in RO for BFO terms that are intended to be defined in RO to have `<rdfs:isDefinedBy rdf:resource="http://purl.obolibrary.org/obo/ro.owl"/>`. Currently we assume `IsDefiningOntology=true` only when prefix of ontology matches prefix of term. Then we can extend this to consider `rdfs:isDefinedBy`.

**Comment by jamesamcl** [comment]
We do consider rdfs:isDefinedBy I think

**Comment by jamesamcl** [comment]
We are using the wrong predicate. This should be changed to isDefinedBy rather than definedBy:

https://github.com/EBISPOT/ols4/blob/dev/dataload/linker/src/main/java/LinkerPass1.java#L238C34-L238C34

**Comment by serjoshua** [comment]
@udp should we just change that line? Will there be any dependencies we need to be aware of and check?

**Comment by haideriqbal** [comment]
Closing this due to inactivity. Feel free to reopen
